### PR TITLE
Sensor updates

### PR DESCRIPTION
**Pull request by rhysfaultless-cpr**

### CODE REVIEW

File: docs/sensors/lidar_2d/hokuyo_ust20_lx.mdx [hunk 1]
```diff
@@ -15,10 +15,9 @@ import Support from "/components/support.mdx";
 
 ## Item Numbers
 
-| Description                      |                         CPR item                         |
-| :------------------------------- | :------------------------------------------------------: |
-| Hokuyo UST20-LX sensor           | [016359](/assets/pdf/clearpath_robotics_016359-TDS1.pdf) |
-| Hokuyo UST20-LX sensor, PACS kit |                                                          |
+| Description            |                         CPR Item                         | Sales Kit | Installation Kit, With Fasteners |
+| :--------------------- | :------------------------------------------------------: | :-------: | :------------------------------: |
+| Hokuyo UST20-LX sensor | [016359](/assets/pdf/clearpath_robotics_016359-TDS1.pdf) |           |                                  |
```
Comment (jhiggins-cpr) [inline docs/sensors/lidar_2d/hokuyo_ust20_lx.mdx]: Assuming that the kit information is intentionally blank here? Should we add a placeholder like "N/A" or "Coming soon"? Up to you.

File: docs/sensors/lidar_2d/hokuyo_ust30_lx.mdx [hunk 2]
```diff
@@ -15,10 +15,9 @@ import Support from "/components/support.mdx";
 
 ## Item Numbers
 
-| Description                      |                         CPR item                         |
-| :------------------------------- | :------------------------------------------------------: |
-| Hokuyo UST20-LX sensor           | [027584](/assets/pdf/clearpath_robotics_027584-TDS1.pdf) |
-| Hokuyo UST20-LX sensor, PACS kit |                                                          |
+| Description            |                         CPR Item                         | Sales Kit | Installation Kit, With Fasteners |
+| :--------------------- | :------------------------------------------------------: | :-------: | :------------------------------: |
+| Hokuyo UST30-LX sensor | [027584](/assets/pdf/clearpath_robotics_027584-TDS1.pdf) |           |                                  |
```
Comment (jhiggins-cpr) [inline docs/sensors/lidar_2d/hokuyo_ust30_lx.mdx]: Same note as above

**Review by jhiggins-cpr — APPROVED** [review]

**Review by jhiggins-cpr — APPROVED** [review]

File: docs/sensors/lidar_2d/hokuyo_ust20_lx.mdx (hunk 1 above)
Comment (rhysfaultless-cpr) [reply docs/sensors/lidar_2d/hokuyo_ust20_lx.mdx]: I have added the kit item numbers for the UST20-LX.

File: docs/sensors/lidar_2d/hokuyo_ust30_lx.mdx (hunk 2 above)
Comment (rhysfaultless-cpr) [reply docs/sensors/lidar_2d/hokuyo_ust30_lx.mdx]: I have added the kit item numbers for the UST30-LX.

**Pull request merged**